compiler/modules/front/stmt.bal

```diff
@@ -41,7 +41,7 @@ type ClosureContext object {
 };
 
 type CapturedRegisterMemo readonly & record {|
-    int captured;
+    bir:CapturableRegister captured;
     bir:CapturedRegister reg;
```

Comment:
Calling this `reg` is not good.

Maybe inner/outer for the two fields.

Reply:
Fixed